### PR TITLE
feat(iconPack): add coolicons

### PR DESCRIPTION
**Pull request by whiterqbbit**
Hi!
I added my fav icon pack. 

I modified the `obsidianExportPath` in the README, as it was at the old name location for me. Maybe that's because I have the plugin installed since an older version ?

Cheers

### CODE REVIEW

File: src/icon-packs.ts [hunk 1]
```diff
@@ -79,6 +79,14 @@ const iconPacks = {
     downloadLink:
       'https://github.com/FlorianWoelki/obsidian-iconize/raw/main/iconPacks/rpg-awesome.zip',
   },
+  /** @source https://coolicons.cool/ */
+  coolicons: {
+    name: 'coolicons',
+    displayName: 'Coolicons',
+    path: '',
+    downloadLink:
+      'https://github.com/FlorianWoelki/obsidian-iconize/raw/main/iconPacks/coolicons.zip',
```
Comment (FlorianWoelki) [inline src/icon-packs.ts:88]: You can probably get this directly from the official github release right?
ref: https://github.com/krystonschwarze/coolicons/releases/tag/v4.1

File: README.md [hunk 2]
```diff
@@ -117,7 +118,7 @@ After the installation, you need to create a `env.js` file in the root directory
 
 ```js
 export const obsidianExportPath =
-  '<path-to-obsidian-vault>/.obsidian/plugins/obsidian-iconize/';
+  '<path-to-obsidian-vault>/.obsidian/plugins/obsidian-icon-folder/';
```
Comment (FlorianWoelki) [inline README.md:121]: You can remove this and keep it as it was.

File: iconPacks/coolicons.zip
Comment (FlorianWoelki) [inline iconPacks/coolicons.zip:1]: This can then also be removed when using the github release directly.

**Review by FlorianWoelki — COMMENTED** [review]
thank you for the pull request. you can feel free to remove the updated path and change the zip file download to the github release by coolicons.

File: src/icon-packs.ts (hunk 1 above)
Comment (whiterqbbit) [reply src/icon-packs.ts]: Absolutely ! I thought it wouldn't work due to a structure with subfolders but it does :+1:

**Comment by FlorianWoelki** [comment]
Thank you for your contribution!

**Pull request merged**